scripts/local-docker-rnode-p2p-test-net-builder.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+# With Docker CE installed, this will build a simple private RChain P2P test network.
+# The test network contains a bootstrap server and two more peers connecting to P2P network via bootstrap.
+set -eo pipefail
+
+NETWORK_UID="1" # Unique identifier for network if you wanted to run multiple test networks
+
+if [[ $1 && $2 ]]; then
+  git_repo=$1
+  branch_name=$2
+  echo "Creating docker rnode test-net for ${git_repo} ${branch_name}"
+else
+  echo "Usage: $0 <repo url>> <branch name>"
+  echo "Usage: $0 https://github.com/rchain/rchain dev"
+  exit
+fi
+
+network_name="${NETWORK_UID}.rnode.test.net"
+
+sudo echo "" # Ask for sudo early
+
+# Create debian package from git repo via sbt
+git_dir=$(mktemp -d /tmp/rchain-git.XXXXXXXX)
+cd ${git_dir}
+git clone ${git_repo} 
+cd rchain
+git checkout ${branch_name}
+sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate node/debian:packageBin
+
+# Create docker network if it doesn't exist
+if [[ "$(sudo docker network list --format {{.Name}} | grep ^${network_name}$)" == "" ]]; then
+  sudo docker network create \
+    --driver=bridge \
+    --subnet=10.1.1.0/24 \
+    --ip-range=10.1.1.0/24 \
+    --gateway=10.1.1.1 \
```

Comment:
You might consider using 169.254.0.0, as it's not routable and can't conflict with the local network setup. You may also consider making the subnet a parameter in the next version, so that people who already have routes to 10.1.1.0/24 don't have a conflict. At some point, though, someone will view this as a bug.